core/src/datayoga_core/blocks/relational/write/block.py

```diff
@@ -1,26 +1,46 @@
 import logging
+from enum import Enum, unique
 from typing import Any, Dict, List, Optional, Tuple
 
 import sqlalchemy as sa
 from datayoga_core import utils, write_utils
 from datayoga_core.block import Block as DyBlock
-from datayoga_core.blocks.relational.connectors import Connector, DbType
 from datayoga_core.context import Context
 from datayoga_core.result import Result
+from sqlalchemy import Table
+from sqlalchemy.engine import CursorResult
+from sqlalchemy.exc import OperationalError
+from sqlalchemy.sql.expression import ColumnCollection
 
 logger = logging.getLogger("dy")
 
 
+@unique
+class DbType(Enum):
+    MSSQL = "mssql"
+    MYSQL = "mysql"
+    PSQL = "postgresql"
+
+    @classmethod
+    def has_value(cls, value: str) -> bool:
+        return value in cls._value2member_map_
+
+
+DRIVERS = {
```

Comment:
what about pg? should we rename to DEFAULT_DRIVERS?

Reply:
done X 2